plugins/reolink/src/main.ts

```diff
@@ -318,10 +324,28 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, DeviceProvider, R
         }
         if (this.hasSiren())
             interfaces.push(ScryptedInterface.DeviceProvider);
+        if (this.hasBattery()) {
+            interfaces.push(ScryptedInterface.Battery, ScryptedInterface.Online);
+            await this.startBatteryCheckInterval();
+        }
 
         await this.provider.updateDevice(this.nativeId, name, interfaces, type);
     }
 
+    async startBatteryCheckInterval() {
```

Comment:
I dont think this function needs to be async?

Reply:
removed